test/build.sh

```diff
@@ -16,8 +16,9 @@ cmakeConfigure()
     local CMAKEARGS=(..
            "-DCMAKE_Fortran_COMPILER=${fcomp}"
            "-DCMAKE_INSTALL_PREFIX=${idir}"
-           "-DCMAKE_CXX_COMPILER=g++"
-           "-DCMAKE_C_COMPILER=gcc"
+           "-DCMAKE_CXX_COMPILER=CC"
+           "-DCMAKE_C_COMPILER=cc"
```

Comment:
CC/cc/gcc/g++ are wrapped in such a way that you get MPI when the correct modules are loaded. We now solve the problem directly within the buildenv where we simply expose `CXX`, `CC` and `FC`. CMake will pick these environment variables up and do whatever it wants with it.